server/appservices/subscriptions.go

```diff
@@ -38,6 +38,34 @@ func (a *AppServices) Subscribe(r *incoming.Request, sub apps.Subscription) erro
 		return err
 	}
 
+	// TODO <>/<> MM-??? deprecate bot_joined|left_...
```

Comment:
I guess we need to create a ticket for this and remove the comment?

Reply:
yes, I actually want to sprinkle the core with the ticket reference to make it easier to remove later.